### PR TITLE
Notifications honza dudek

### PR DESCRIPTION
**Pull request by HonzaDudek**

### CODE REVIEW

File: src/app/_helpers/authentication/auth.guard.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { Injectable } from '@angular/core';
 import { CanActivate, Router } from '@angular/router';
 import { AuthService } from './auth.service';
 import { environment } from '../../../environments/environment';
+import {ENDPOINTS} from '../../../environments/endpoints';
```
Comment (balazsbarni) [inline src/app/_helpers/authentication/auth.guard.ts]: `{ ENDPOINTS }`

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <app-header></app-header>
 <app-alert></app-alert>
-<router-outlet></router-outlet>
+  <router-outlet></router-outlet>
```
Comment (balazsbarni) [inline src/app/app.component.html]: why?

File: src/app/game/notifications/notifications.component.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+})
+export class NotificationsComponent implements OnInit {
+
+  notifications: TribesNotification[];
+  notification: TribesNotification;
+
+  constructor() { }
+
+  ngOnInit() {
+    this.notifications = new NotificationsResponseMock().notifications;
```
Comment (balazsbarni) [inline src/app/game/notifications/notifications.component.ts]: this shouldn't be aquired by the `notifications.service`?

File: src/app/game/notifications/notifications.component.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  width: 90%;
+  margin: 10px auto;
+  background: white;
+  height: 100px;
+
+  display: flex;
+  flex-direction: row;
+}
+
+
```
Comment (balazsbarni) [inline src/app/game/notifications/notifications.component.css:35]: empty lines

File: src/app/game/notifications/notification/notification.component.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  @Input() notification: TribesNotification;
+  completion: number;
+
+  constructor() {
+
+  }
+
+  ngOnInit() {
+    console.log(Date.now());
```
Comment (balazsbarni) [inline src/app/game/notifications/notification/notification.component.ts]: remove the `console.log` please

File: src/app/game/game.component.ts [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
+import {Component, OnInit, ViewChild, OnChanges, SimpleChanges} from '@angular/core';
+import {BuildingsComponent} from './buildings/buildings.component';
+import {NotificationsService} from './notifications/notifications.service';
+import {NotificationsComponent} from './notifications/notifications.component';
 
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.css'],
+  providers: [NotificationsComponent],
```
Comment (balazsbarni) [inline src/app/game/game.component.ts]: no need for providers

File: src/app/game/notifications/notification/notification.component.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    this.getCompletion();
+    setInterval(() => {
+      this.getCompletion();
+      console.log(this.completion);
```
Comment (balazsbarni) [inline src/app/game/notifications/notification/notification.component.ts]: here too

**Review by balazsbarni — CHANGES_REQUESTED** [review]

File: src/app/game/notifications/notifications.component.css (hunk 4 above)
Comment (HonzaDudek) [reply src/app/game/notifications/notifications.component.css:35]: Empty lines are for structuring purposes. They divide display, size and styling properties.

**Review by balazsbarni — APPROVED** [review]

**Pull request merged**